### ISSUE TITLE
Content Creation Documentation

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/uls#17]
We need to create documentation for staff on uploading content and various specifics with content types that we should be aware of when creating content.

**Comment by manolobevia** [comment]
Also inform them how to make content accessible.

**Comment by kevinkidwell** [comment]
Here is an updated version of the content creation documentation. This will be supplemented with screenshots, especially for Elementor.

----

### Content Creation Documentation

**Login/Account**
- Credentials
- URL
	- http://dev-physicalsciences.pantheonsite.io/wp-admin/
- Capabilities of account
	- https://codex.wordpress.org/Roles_and_Capabilities
	- Users who are contributing to pages (not just posts) will have the role of Editor. Those who will only be adding posts will have the role of Author.
	- The Editor role allows users to create and edit any of the pages and posts within the unit site. It also allows them to publish them but before publishing it should still go through the content review workflow.
	- The Author role allows users to create and edit posts within the unit site. These users can only edit or delete their own posts.


**Adding content**
- Difference between pages and posts
	- Pages will be the bulk of the content for a site – all of the pages that users will navigate to for content
	- Posts are used for blog-type sections and for organizing content such as databases by tags.
- Adding New
	- Once logged in, in the left-hand sidebar, go to Pages or Posts and then Add New
	- Tagging
		- Use the Tags in the right hand column on the Post/Page to add the proper tag
		- Tagging takes place before going to Elementor, if tags are necessary
	- Add title and then select Edit with Elementor
- Using Elementor for editing
	- Templates are defined for layouts
		- To add a template, click on the folder icon in the body of the page, navigate to My Templates, and select a template that matches the layout necessary
	- Content should always be edited in Elementor so that it is responsive and works with pre-built code
	- Content editing
		- Text
			- Editing text will appear in the left hand column when text is selected. This is where all text should be edited.
			- If you are adding links, those options will also appear in the left hand column
		- Images
			- Image options will appear in the left hand column when an image or image placeholder is selected
- Making content accessible
	- Follow Accessibility review section below


**Reviewing content**
- Workflow for review
	- Creator will create the page/post and then send it to the content reviewer
	- Content Reviewer will review the content that is sent to them. If there are any content edits, they can do it themselves or send back to Creator
	- Accessibility Reviewer will review the accessibility of content send to them. This role will ideally be filled by the Content Reviewer with a little training in accessibility practices. They can edit or send back to Creator for edits.
	- Once the Accessibility Reviewer says content is good to go, it is approved and published.
- Accessibility review
	- Guidelines – WCAG 2.0
		- https://webaim.org/standards/wcag/checklist
		- Must be AA compliant
	- Images
		- Make sure images do not have text on them that is important (screen readers cannot read images)
		- Any text on images must be decorative only
		- Images must have alt text
	- Videos
		- Any video multimedia must have captions
		- More info and training available on video accessiblity
	- Links
		- Link text should be relevant to the link (not “click here”)
		- Links must be differentiated from surrounded text and follow contrast – (if text around link is regular, it should be bold, italic, or underlined along with the color for links within theme)
	- Color contrast
		- Colors need to follow the proper contrast guidelines
		- https://webaim.org/resources/contrastchecker/
	- Resources
		- IT PR1646 Web Accessibility for Content Contributors – Recommended Course
		- SiteImprove
		- WebAIM WAVE


- Adding to navigation (where content belongs in nav and how to add it)
	- Items should only be added to the navigation menu if it is a top-level accessible page and only once it has been approved and published by the reviewer(s)
	- Pages get added to the navigation in the form of adjusting the Menu
	- In the left-hand sidebar, go to Appearance > Menus
	- Here you will need to select the proper menu (if multiple)
	- You can use the left box on this page to select what you want to add to the navigation by checking the box and clicking Add to Menu
	- After it is added to the menu, scroll to the bottom of the list in the box on the right and then drag the new item to its proper place within the menu structure
	- Once it is correct, select Save Menu

**Comment by mhk33** [comment]
As mentioned in the stand-up this morning, one item that came up in the central-IT accessibility review was certain Vet pages with the collapse UI (e.g. https://vet.library.cornell.edu/library-collections/) had both h4 and strong tags applied. The recommendation was to remove <strong> and add bold styles to the h4 (which we already have). 

The affected pages had bold applied in the editor. I removed the bold for those pages.

Just making note of this if we want to include documentation on building/editing the collapse UI, although I understand this may be built differently in Elementor.

**Comment by kevinkidwell** [comment]
@mhk33 I updated my comment above to add in notes about both of those. I have a feeling for accessibility one of us is going to have to review the content at first to make sure that it's good. It would be nice if we can find a way to have all content creators go through one of the accessibility courses or be able to teach them and have that review on their end.

**Comment by mhk33** [comment]
@kevinkidwell Agreed. I am happy to help review content. We'll have to figure out a way to encourage them to go to training and also might have to be the police for a while, and point out what to do and not to do. I just did that on the current main site where someone used an URL for link text (where they really did not need to).

**Comment by kevinkidwell** [comment]
[ULS_Units_Content_Guide.pdf](https://github.com/cul-it/uls/files/2876814/ULS_Units_Content_Guide.pdf)

**Comment by mhk33** [comment]
This is great!

What do you think about creating an online style guide as well? I think we discussed options for that, but maybe it was on another issue.
